README.md

```diff
@@ -296,7 +299,7 @@ On the open source side, there are many related projects. [php-mf2-shim](https:/
 
 Changelog
 ---
-### 3.0 - unreleased
+### 3.0 - 2020-03-29
```

Comment:
mind updating this to the date you think you'll be ready to finish the release and push it to pypi? can be today, or in the future. (and then run `./docs/build.sh` again.)